.github/workflows/test.yml

```diff
@@ -0,0 +1,106 @@
+name: test
+on:
+  pull_request:
+  push:
+    branches:
+      - "master"
```

Comment:
Why restrict the branch?

When I pushed a new branch to my fork and I want to run test, I need to edit this file.
It is cumbersome.

Reply:
it comes from starter workflows https://github.com/actions/starter-workflows/blob/c59b62dee0eae1f9f368b7011cf05c2fc42cf084/ci/go.yml#L3-L7
but I agree with you, and will remove it.